icu-date.lua

```diff
@@ -1,9 +1,12 @@
 local ffi = require("ffi")
 local detect_icu_version_suffix = require("icu-date.detect_icu_version_suffix")
 local icu_ffi_cdef = require("icu-date.ffi_cdef")
+local box = require("local")
 
 local icu_version_suffix = detect_icu_version_suffix()
-icu_ffi_cdef(ffi, icu_version_suffix)
+box.once('icu-date:init', function()
```

Comment:
So at the second run you will not have needed ffi.C.* functions.

Reply:
Hm.. Then how to make a package reloading?

`package.reload(x)`

Reply:
@knazarov Can you suggest something here?

Reply:
@Totktonada what about `tcall`?

https://www.lua.org/pil/8.4.html

Reply:
Whether you just want to check whether these functions are defined?

```
ffi = require('ffi')
ffi.C.kill
---
- error: '[string "return ffi.C.kill"]:1: missing declaration for symbol ''kill'''
...
ffi.cdef([[ void kill(int, int); ]])
ffi.C.kill
---
- 'cdata<void ()>: 0x7fce7202a890'
...
```

So, yep, pcall() does the trick. Choose one of defined functions and try to acquire it under pcall().

Reply:
I want to escape repeated call of `cdef`.
Tomorrow I'll try pcall

Reply:
@Totktonada check new implementation

Reply:
Re:
@Totktonada check new implementation

p.s. my dev server waiting for you)